dashboard/test/ui/features/level_group_multi_page_dots.feature

```diff
@@ -0,0 +1,70 @@
+@no_mobile
+@no_circle
+@as_student
+Feature: Level Group
+
+Background:
+  Given I am on "http://learn.code.org/s/allthethings/stage/22/puzzle/2/page/1?noautoplay=true&force_submittable=true"
+  Then I rotate to landscape
+  And I wait to see ".nextPageButton"
+  And element ".nextPageButton" is visible
+
+Scenario: Submit three pages as... 1. some, 2. none, 3. all questions answered.
+  When element ".level-group-content:nth(0) .multi-question" contains text "Which arrow gets"
+
+  # Enter answers to all three multis on the first page.
+  And I press ".level-group-content:nth(0) .answerbutton[index=2]" using jQuery
+  And I press ".level-group-content:nth(1) .answerbutton[index=1]" using jQuery
+  # Pressing 1, 2, 0 should result in 2, 0 being selected.
+  And I press ".level-group-content:nth(2) .answerbutton[index=1]" using jQuery
+  And I press ".level-group-content:nth(2) .answerbutton[index=2]" using jQuery
+  And I press ".level-group-content:nth(2) .answerbutton[index=0]" using jQuery
+
+  And I press ".nextPageButton" using jQuery
+  And I wait for 30 seconds
```

Comment:
Can we wait to see specific elements instead of a fixed timeout?  30 seconds seems like a long time to wait.